cmd/api/generic.go

```diff
@@ -16,7 +16,7 @@ func makeTxRoute(router gin.IRouter, api blockatlas.Platform) {
 		return
 	}
 
-	router.GET("/:address", func(c *gin.Context) {
+	router.GET("/transactions/:address", func(c *gin.Context) {
```

Comment:
@vikmeup Moving the transactions route is quite a breaking change. How about doing it on a `/v2` route instead?

Reply:
`v2` sounds good!